turbo/templatetags/turbo_streams_helpers.py

```diff
@@ -0,0 +1,16 @@
+from django import template
+from django.core.signing import Signer
+
+from turbo import channel_name_for_instance
+
+register = template.Library()
+
+# https://docs.djangoproject.com/en/3.1/topics/signing/
+
+
+@register.inclusion_tag('turbo/turbo_stream_source.html')
+def turbo_stream_from(model_instance):
+    signer = Signer()
```

Comment:
For a bit of extra security here it might be good to add a Turbo-specific salt (https://docs.djangoproject.com/en/3.1/topics/signing/#using-the-salt-argument). Also, it might be a good idea to specify the `algorithm` argument. I've been bitten when Django changes defaults between versions, especially around crypto. At the least it will help keep tests from breaking. A downside is that `algorithm` was only added in Django 3.1. Which means they expect to change it, but also makes supporting < 3.1 difficult.

Reply:
I'm not sure if a Turbo-specific salt will actually be more secure, since then every instance of `message-1` for example will still have the same signature. I could be wrong, but I thought that salts are mostly effective for if you store the salt alongside the hashed data and had a unique one per-row so that make rainbow tables couldn't be used to crack your hash. I could be wrong though, and I'll look more into it!

Reply:
My understanding is the same as yours @davish: a static salt is not effective, only having a different salt for every stream would strengthen our security (and that seems a bit too much effort, at least for now)

Reply:
The (admittedly small) benefit to having a Turbo salt is to avoid clients using strings generated by the application, but outside of the Turbo package, from being used as Turbo payloads. It just scopes all signatures to this package only. This may just be my PTSD from working on large Django code bases in the past though. 

It would also be cool to use some kind of session/user ID as the salt too, but right, that would be much more work.